### PR TITLE
Nbo/disable select scenario while edition mode prod 7591

### PR DESCRIPTION
**Pull request by Nikeleos**

### CODE REVIEW

**Review by COSMO-VMI — APPROVED** [review]
LGTM, GG!

File: src/components/CreateScenarioDialog/CreateScenarioButton.js [hunk 1]
```diff
@@ -41,9 +43,20 @@ const CreateScenarioButton = ({
 
   return (
     <div>
-      <Button size="medium" startIcon={<AddIcon />} variant="text" onClick={openDialog} color="primary">
-        <Typography noWrap color="primary">{t('commoncomponents.button.create.scenario', 'Create Alternative Scenario')}</Typography>
-      </Button>
+      <Tooltip arrow title={t(buttonTooltip)}>
```
Comment (jreynard-code) [inline src/components/CreateScenarioDialog/CreateScenarioButton.js:46]: Could you please add default wording?

File: src/components/HierarchicalComboBox/HierarchicalComboBox.js [hunk 2]
```diff
@@ -25,25 +34,25 @@ const HierarchicalComboBox = ({ tree, label, disabled, handleChange, separator,
...
-        </React.Fragment>
       )}
       renderInput={(params) => (
-        <TextField
-          {...params}
-          placeholder={t(label)}
-          label={t(label)}
-          variant="outlined"
-        />
+        <Tooltip arrow title={t(renderInputToolType)}>
```
Comment (jreynard-code) [inline src/components/HierarchicalComboBox/HierarchicalComboBox.js:48]: Could you please add default wording ?

File: src/components/HierarchicalComboBox/HierarchicalComboBox.js [hunk 3]
```diff
@@ -25,25 +34,25 @@ const HierarchicalComboBox = ({ tree, label, disabled, handleChange, separator,
...
+          <TextField
+            {...params}
+            placeholder={t(label)}
```
Comment (jreynard-code) [inline src/components/HierarchicalComboBox/HierarchicalComboBox.js:51]: Could you please add default wording ?

File: src/components/HierarchicalComboBox/HierarchicalComboBox.js [hunk 4]
```diff
@@ -25,25 +34,25 @@ const HierarchicalComboBox = ({ tree, label, disabled, handleChange, separator,
...
+            label={t(label)}
```
Comment (jreynard-code) [inline src/components/HierarchicalComboBox/HierarchicalComboBox.js:52]: Could you please add default wording ?

File: src/views/Scenario/Scenario.js [hunk 5]
```diff
@@ -65,9 +65,12 @@ const Scenario = (props) => {
   const workspaceId = workspace.data.id;
   const [editMode, setEditMode] = useState(false);
 
-  function handleScenarioChange (event, scenario) {
+  const currentScenarioRenderInputToolType = editMode ? 'views.scenario.dropdown.scenario.tooltip.disabled' : '';
```
Comment (jreynard-code) [inline src/views/Scenario/Scenario.js:68]: Could you please extract 'views.scenario.dropdown.scenario.tooltip.disabled' into a Constant file?

File: src/views/Scenario/Scenario.js [hunk 6]
```diff
@@ -65,9 +65,12 @@ const Scenario = (props) => {
...
+  const createScenarioButtonToolType = editMode ? 'commoncomponents.button.create.scenario.tooltip.disabled' : '';
```
Comment (jreynard-code) [inline src/views/Scenario/Scenario.js:69]: Could you please extract 'commoncomponents.button.create.scenario.tooltip.disabled' into a Constant file?

**Review by jreynard-code — CHANGES_REQUESTED** [review]
LGTM, just minor changes

File: src/views/Scenario/Scenario.js (hunk 5 above)
Comment (Nikeleos) [reply src/views/Scenario/Scenario.js:68]: Where it is done elsewhere in the code ???

File: src/views/Scenario/Scenario.js (hunk 6 above)
Comment (Nikeleos) [reply src/views/Scenario/Scenario.js:69]: Wont do. It must be done in one separated ticket

File: src/views/Scenario/Scenario.js (hunk 5 above)
Comment (Nikeleos) [reply src/views/Scenario/Scenario.js:68]: Wont do. It must be done in one separated ticket

**Pull request merged**